cmd/admin-router.go

```diff
@@ -36,6 +37,22 @@ const (
 // adminAPIHandlers provides HTTP handlers for MinIO admin API.
 type adminAPIHandlers struct{}
 
+// adminApiHostHandler - allow access to the  Admin APIs only from local interface by default.
+func adminApiHostHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		host := strings.Split(r.Host, ":")[0]
+		if host == "localhost" || host == "127.0.0.1" {
```

Comment:
If I understand correctly, this verifies that the `Host:` header of the HTTP request is either `localhost` or `127.0.0.1`. I am no HTTP expert, but isn't it possible to send an HTTP request from a remote machine and set the `Host:` header to `localhost` anyway?

Reply:
Agree, updated